lib/streamlit/commands/page_config.py

```diff
@@ -142,3 +174,51 @@ def get_random_emoji():
     # Weigh our emojis 10x, cuz we're awesome!
     # TODO: fix the random seed with a hash of the user's app code, for stability?
     return random.choice(RANDOM_EMOJIS + 10 * ENG_EMOJIS)
+
+
+def set_menu_items_proto(lowercase_menu_items, menu_items_proto):
+    if "get help" in lowercase_menu_items:
+        if lowercase_menu_items["get help"] is not None:
+            menu_items_proto.get_help_url = lowercase_menu_items["get help"]
+        else:
+            menu_items_proto.hide_get_help = True
+
+    if "report a bug" in lowercase_menu_items:
+        if lowercase_menu_items["report a bug"] is not None:
+            menu_items_proto.report_a_bug_url = lowercase_menu_items["report a bug"]
+        else:
+            menu_items_proto.hide_report_a_bug = True
+
+    if "about" in lowercase_menu_items:
+        if lowercase_menu_items["about"] is not None:
+            menu_items_proto.about_section_md = dedent(lowercase_menu_items["about"])
+
+
+def validate_menu_items(dict):
+    for k, v in dict.items():
+        if not valid_menu_item_key(k):
+            raise StreamlitAPIException(
+                "We only accept the keys: "
+                f'"Get help", "Report a bug", and "About" ("{k}" is not a valid key.)'
+            )
+        if v is not None:
+            if not valid_url(v) and k != "about":
+                raise StreamlitAPIException(f'"{v}" is a not a valid URL!')
+
+
+def valid_menu_item_key(key):
+    return key == "get help" or key == "report a bug" or key == "about"
```

Comment:
two nits: It'd be slightly cleaner to add some consts
```python
GET_HELP_KEY = "get help"
REPORT_A_BUG_KEY = "report a bug"
ABOUT_KEY = "about"
```

and use those keys everywhere hardcoded strings are currently so that things are easier to change in the future.

The second nit is that this reads very slightly more nicely as
```python
return key in [GET_HELP_KEY, REPORT_A_BUG_KEY, ABOUT_KEY]:
```